qubes-artwork.spec.in

```diff
@@ -55,27 +50,33 @@ Summary:    Qubes OS theme for EFI boot manager (rEFInd)
 %description efi
 Qubes OS theme for rEFInd boot manager
 
+%package anaconda
```

Comment:
What will pull in this new package?

Reply:
This https://github.com/QubesOS/qubes-lorax-templates/pull/7 and https://github.com/QubesOS/qubes-qubes-release/pull/9. Forgot to PR them. A submodule update in installer is thus needed.